LanguageFeatures/Wildcards/unchanged_A05_t10.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Other declarations: top-level variables, top-level function
+/// names, type names, member names, etc. are unchanged. They can be named `_`
+/// as they are today.
+///
+/// @description Checks that it is a compile-time error to declare more than one
+/// member named `_`. Test an instance variable vs other class members conflict.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=wildcard-variables
+
+class C1 {
+  int _ = 1;
+  static int _ = 1;
+//           ^
+// [analyzer] unspecified
+// [cfe] unspecified
+}
```

Comment:
Isn't this a duplicate of something like `C5` in LanguageFeatures/Wildcards/unchanged_A05_t06.dart?

Reply:
Perhaps the easiest way to avoid duplicates would be to have instance/static clashes in their own tests (like this one) and omit the instance/static clashes in those other tests (like A05_t06)? I'm not quite sure which approach is easier, but it does look like the current system will generate some duplicates.

(In some cases, e.g., augmenting declarations, it does make sense to test different orderings of elements; but in general we'll just have to assume that it has already been tested "somewhere else" that different textual orderings of declarations will give rise to the same name clash errors, otherwise everything will explode ;-).